modules/core/src/main/scala/fs2/kafka/KafkaConsumer.scala

```diff
@@ -554,6 +547,10 @@ object KafkaConsumer {
 
       override def toString: String =
         "KafkaConsumer$" + id
+
+      override def terminate: F[Unit] = fiber.cancel
+
+      override def awaitTermination: F[Unit] = fiber.join
```

Comment:
Perhaps this should be 
```suggestion
      override def terminate: F[Unit] = polls.join *> actor.join
```
or
```suggestion
      override def terminate: F[Unit] = polls.join.attempt.void *> actor.join.attempt.void
```

It just means that the shutdown is ordered. What do you think?

Reply:
I think that would swallow errors? Again, probably better to start by inlining the combine operation on Fiber. But I think that can be a separate PR.